### PR TITLE
Add option to disable remote control

### LINKED ISSUE #601: Add option to disable remote control
The remote control feature is slightly scary from a security perspective. People will use this addon to connect to public or shared Jellyfin servers, and Jellyfin is not the most secure piece of software to begin with ([eg](https://github.com/jellyfin/jellyfin/issues/5415)). Allowing untrusted servers to simulate button presses creates a backdoor into people's home networks

It would be fantastic if a setting could be added to disable all forms of remote control, and only use the websocket connection for notifications and library update signals

### PR DESCRIPTION
**Pull request by mcarlton00**
Untested, because my Kodi is still fubared and keeps imploding every time I open the addon settings page, but should work.  Probably.

### CODE REVIEW

File: jellyfin_kodi/jellyfin/ws_client.py [hunk 1]
```diff
@@ -79,21 +79,23 @@ def on_error(self, ws, error):
...
-                "SetAudioStreamIndex,SetSubtitleStreamIndex,"
-                "SetRepeatMode,"
-                "Mute,Unmute,SetVolume,"
-                "Play,Playstate,PlayNext,PlayMediaSource"
-            ),
-        })
+        if settings('remoteControl.bool'):
+            self.client.jellyfin.post_capabilities({
+                'PlayableMediaTypes': "Audio,Video",
+                'SupportsMediaControl': True,
```
Comment (oddstr13) [inline jellyfin_kodi/jellyfin/ws_client.py:85]: Probably want to still send this, with `SupportsMediaControl` False and empty `SupportedCommands`?

Or is capabilities exclusively used for remote control and not media?

After this we'd still accept and act on remote control commands, if the server is made to ignore the data sent here in some way.

Comment (mcarlton00) [reply jellyfin_kodi/jellyfin/ws_client.py:85]: I tried just sending `{'SupportsMediaControl': False}` and the server laughed at me and kept allowing remote control anyway.  I can't say for certain if that's the actual results or if it was just a quirk of my kodi instance crashing every 30 seconds though.  Sending an empty payload was the only reliable way I had it stop showing up in the play to menu.

Comment (oddstr13) [reply jellyfin_kodi/jellyfin/ws_client.py:85]: Well, if playback still works with this, then it's fine.

Comment (mcarlton00) [reply jellyfin_kodi/jellyfin/ws_client.py:85]: I can't test any of it.  If you want to test flipping the payload around and seeing if there's any difference then cool, but I'm dev-ing blind here until I figure out what's causing this addon to hard crash Kodi on my workstation.

**Review by oddstr13 — APPROVED** [review]
I think this PR only partially addresses #601, as it doesn't touch the code interpreting the commands sent from the server.

In theory it would still be possible to use remote control by having the server ignore the provided capabilities. This would however require a modified server.

**Pull request merged**